pbsmetrics/metrics_mock.go

```diff
@@ -52,6 +53,15 @@ func (me *MetricsEngineMock) RecordAdapterRequest(labels AdapterLabels) {
 	me.Called(labels)
 }
 
+// RecordAdapterConnections mock
+func (me *MetricsEngineMock) RecordAdapterConnections(bidderName openrtb_ext.BidderName, connSuccess bool, info httptrace.GotConnInfo, obtainConnectionTime time.Duration) {
+	me.Called(bidderName, connSuccess, info, obtainConnectionTime)
+}
+
+// RecordDNSTime mock
+func (me *MetricsEngineMock) RecordDNSTime(dnsLookupTime time.Duration) {
+}
```

Comment:
Why don't we call the `Called` function for this method? Looks like something you might have missed by mistake?

Reply:
Not necessary for `TestRecordDNSTime`

Reply:
I still recommend to add the calls, so future tests can rely on that behavior if needed in the future.